config.dev.toml

```diff
@@ -306,3 +307,16 @@ SingleFlightType = "memory"
         # Name of container in the blob storage
         # Env override: ATHENS_AZURE_CONTAINER_NAME
         ContainerName = "MY_AZURE_BLOB_CONTAINER_NAME"
+
+    [Storage.HTTP]
+        # The root URL into which modules will be uploaded
```

Comment:
nit: will they be downloaded from this URL as well? If so, can you add that here?